src/views/ApplicationPage/index.js

```diff
@@ -16,6 +16,17 @@ import SubmitButton from "../../components/SubmitButton";
 
 import styles from "./styles.module.css";
 
+var AWS = require("aws-sdk");
+
+let awsConfig = {
+  region: "us-east-1",
+  endpoint: "http://dynamodb.us-east-2.amazonaws.com",
+  accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY,
+  secretAccessKey: process.env.REACT_APP_AWS_SECRET_KEY,
+};
+
+AWS.config.update(awsConfig);
+
 export default class application extends Component {
```

Comment:
capitalize "Application" for consistency

Reply:
Done!